core/conversion/converters/impl/element_wise.cpp

```diff
@@ -48,775 +48,755 @@ nvinfer1::ITensor* scalar_to_tensor(ConversionCtx* ctx, at::Scalar s) {
-
-               LOG_DEBUG("Output tensor shape: " << out->getDimensions());
-               return true;
-             }})
-        .pattern(
-            {"aten::le.Tensor(Tensor self, Tensor other) -> (Tensor)",
-             [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
-               auto self = args[0].ITensorOrFreeze(ctx);
-               auto other = args[1].ITensorOrFreeze(ctx);
-
-               auto less = add_elementwise(
-                   ctx, nvinfer1::ElementWiseOperation::kLESS, self, other, util::node_info(n) + "_less");
-               TORCHTRT_CHECK(less, "Unable to create Less layer from node: " << *n);
-
-               auto equal = add_elementwise(
-                   ctx, nvinfer1::ElementWiseOperation::kEQUAL, self, other, util::node_info(n) + "_equal");
-               TORCHTRT_CHECK(equal, "Unable to create Equal layer from node: " << *n);
-
-               auto or_op = ctx->net->addElementWise(
-                   *less->getOutput(0), *equal->getOutput(0), nvinfer1::ElementWiseOperation::kOR);
-
-               TORCHTRT_CHECK(or_op, "Unable to create Or layer from node: " << *n);
-               or_op->setName(util::node_info(n).c_str());
-               auto out = ctx->AssociateValueAndTensor(n->outputs()[0], or_op->getOutput(0));
-
-               LOG_DEBUG("Output tensor shape: " << out->getDimensions());
-               return true;
-             }})
-        .pattern(
-            {"aten::le.Scalar(Tensor self, Scalar other) -> (Tensor)",
-             [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
-               auto self = args[0].ITensorOrFreeze(ctx);
-               auto other = scalar_to_tensor(ctx, args[1].unwrapToScalar());
-               if (self->getType() != other->getType()) {
-                 other = castITensor(ctx, other, self->getType());
-               }
-
-               auto less = add_elementwise(
-                   ctx, nvinfer1::ElementWiseOperation::kLESS, self, other, util::node_info(n) + "_less");
-               TORCHTRT_CHECK(less, "Unable to create Less layer from node: " << *n);
-
-               auto equal = add_elementwise(
-                   ctx, nvinfer1::ElementWiseOperation::kEQUAL, self, other, util::node_info(n) + "_equal");
-               TORCHTRT_CHECK(equal, "Unable to create Equal layer from node: " << *n);
-
-               auto or_op = ctx->net->addElementWise(
-                   *less->getOutput(0), *equal->getOutput(0), nvinfer1::ElementWiseOperation::kOR);
-
-               TORCHTRT_CHECK(or_op, "Unable to create Or layer from node: " << *n);
-               or_op->setName(util::node_info(n).c_str());
-               auto out = ctx->AssociateValueAndTensor(n->outputs()[0], or_op->getOutput(0));
-
-               LOG_DEBUG("Output tensor shape: " << out->getDimensions());
-               return true;
-             }});
+        .pattern({"aten::add.Tensor(Tensor self, Tensor other, Scalar alpha=1) -> "
```

Comment:
Looks like these are linter changes. Or has anything changed ? 
Can you force  checkout master for this  file ?

Reply:
There are real changes in this file as well. Let me see if I can clean it up.

Reply:
Should be more clear now @peri044

Reply:
Thanks !!